content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -447,30 +447,10 @@ To switch from a local encryption provider to the `kms` provider and re-encrypt
    kubectl get secrets --all-namespaces -o json | kubectl replace -f -
    ```
 
-## Disabling encryption at rest
+## {{% heading "whatsnext" %}}
 
-To disable encryption at rest:
+<!-- preserve legacy hyperlinks -->
+<a id="disabling-encryption-at-rest" />
 
-1. Place the `identity` provider as the first entry in the configuration file:
-
-   ```yaml
-   apiVersion: apiserver.config.k8s.io/v1
-   kind: EncryptionConfiguration
-   resources:
-     - resources:
-         - secrets
-       providers:
-         - identity: {}
-         - kms:
-             apiVersion: v2
-             name : myKmsPlugin
-             endpoint: unix:///tmp/socketfile.sock
-   ```
-
-1. Restart all `kube-apiserver` processes.
-
-1. Run the following command to force all secrets to be decrypted.
-
-   ```shell
-   kubectl get secrets --all-namespaces -o json | kubectl replace -f -
-   ```
+If you no longer want to use encryption for data persisted in the Kubernetes API, read
```

Comment:
👋 @sftim .
The changes look okay. The task page that you link to does not provide step-by-step instructions,
[Decrypt Confidential Data that is Already Encrypted at Rest](https://deploy-preview-44086--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/decrypt-data/#configure-the-api-server-to-decrypt-objects).

Reply:
The right hand nav of https://deploy-preview-44086--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/decrypt-data/#decrypting-all-data looks like step by step instructions. Some context is missing; for example, we don't mention making a connection using SSH because we don't actually know that your control plane nodes use SSH for remote access.

If we should clarify that task, we could file an issue about improving that task page &mdash; would you be willing to write up a feature request @kbhawkey?